packages/dev/core/src/Meshes/abstractMesh.ts

```diff
@@ -536,10 +536,35 @@ export abstract class AbstractMesh extends TransformNode implements IDisposable,
      */
     public alphaIndex = Number.MAX_VALUE;
 
+    /**
+     * If set to true, a mesh will only be visible only if its parent(s) are also visible (default is false)
+     */
+    public inheritedVisibility = false;

```

Comment:
Can you stock it on our property bag to avoid increasing the number of class members?

Reply:
of course. let me do that now